filebeat/module/traefik/_meta/docs.asciidoc

```diff
@@ -1,3 +1,60 @@
+:modulename: traefik
+
 == Traefik module
 
-This is the traefik module.
+The +{modulename}+ module parses access logs created by
+https://traefik.io/[Træfik].
+
+include::../include/what-happens.asciidoc[]
+
+[float]
+=== Compatibility
+
+This module requires the
+{elasticsearch-plugins}/ingest-user-agent.html[ingest-user-agent] and
+{elasticsearch-plugins}/ingest-geoip.html[ingest-geoip] Elasticsearch plugins.
+
+//REVIEWERS: Do we need to say anything else about compatibility here?
```

Comment:
I can't think of anything else. I can't tell what version of Traefik was the source of the test logs.